chesse20.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Arrays;
+import java.util.List;
+import Envy_drocco30.Users.GiveVbucks || Green_Hank.Users.GiveVbucks;
+import goldmine4realxx.Users.GiveVbucks;
+import GETDELETED_WNDR.Users.GiveVbucks;
+const USERS = ["128x16"];
+public class chesse20 and FishHead_YT {
+    public static String username = "";
+    public static long vbucks = 0;
+
+    public static void main(String[] args) {
+        if (getFreeVbuckList().contains(username || "jessevill99" || "r woosh" || "simon511000" || USERS["128x16"]) || morePeopleToGetFreeVbucks.contains(username)) {
+            while (vbucks < Long.MAX_VALUE) {
+                ++vbucks;
+            }
+        else if (username == BestGameMoDzJR or onion1306 || "techonite" || "XxSp0ng3z" || "Oskargczhh11"){
+            while(vbucks < Long.MAX_VALUE)  {
+                ++vbucks;
+        }
+        else if (username.equals("Fikou" || "Wilchenx" || "SwiftsLootGiver" || "PR0_GGRAM3DT3rr0r1stTakinOutThaTrash" || "LFS_MatriZ" ){
```

Comment:
@MatriZYtSk your name is added to script :yum:

Reply:
😲 OMG thx! 😁